dot/sync/interface.go

```diff
@@ -15,7 +15,7 @@ import (
 	"github.com/libp2p/go-libp2p-core/peer"
 )
 
-//go:generate mockery --name BlockState --structname BlockState --case underscore --keeptree
+//go:generate mockgen -destination=mock_interface_test.go -package=$GOPACKAGE . BlockState,StorageState,CodeSubstitutedState,TransactionState,BabeVerifier,FinalityGadget,BlockImportHandler,Network
```

Comment:
:+1: :+1: Congrats on the migration!

Although please remove the mocks directory containing the older mockery mocks. If you can't do it in the scope of this PR, keep the older mockery `//go:generate` commands and comment above them stating why they are kept for now together with mockgen mocks.

Reply:
The older mockery mocks are still being used, I added TODO comment.

Reply:
Then let's keep the older mockery //go:generate commands I think

Reply:
Please remove unused mockery mock files in `dot/sync/mocks`